### PR TITLE
Tx insights fixes

### PR DESCRIPTION
**Pull request by danjm**
Fixes issues Dec 2: 1, 2 and 3, and Dec 6: 5 and 6, in the v10.8.0 RC doc

### CODE REVIEW

File: ui/components/app/transaction-list-item-details/transaction-list-item-details.component.js [hunk 1]
```diff
@@ -252,15 +252,18 @@ export default class TransactionListItemDetails extends PureComponent {
...
-                  transactionGroup={transactionGroup}
-                  className="transaction-list-item-details__transaction-activity-log"
-                  onCancel={this.handleCancel}
-                  onRetry={this.handleRetry}
-                  isEarliestNonce={isEarliestNonce}
-                />
-              </Disclosure>
+              {transactionGroup.initialTransaction.type !==
+                TRANSACTION_TYPES.INCOMING && (
+                <Disclosure title="Activity log" size="small">
```
Comment (darkwing) [inline ui/components/app/transaction-list-item-details/transaction-list-item-details.component.js:257]: Nit: needs localization

**Review by darkwing — CHANGES_REQUESTED** [review]

File: ui/components/app/transaction-list-item-details/transaction-list-item-details.component.js (hunk 1 above)
Comment (danjm) [reply ui/components/app/transaction-list-item-details/transaction-list-item-details.component.js]: fixed in bad0de1ce

**Review by darkwing — DISMISSED** [review]

**Review by adonesky1 — DISMISSED** [review]
LGTM!

**Review by georgewrmarshall — DISMISSED** [review]
LGTM! Read through [doc](https://docs.google.com/document/d/1vy-i_UdG4SbmmKenXwY6ih30lAuFU6SQfRaG_QmKjAc/edit#) and have tested that:

1. Empty activity log is hidden on receive transaction ✅ 
Screen Shot 2021-12-07 at 2 00 10 PM

2. Correct method name shown ✅ 
Screen Shot 2021-12-07 at 2 03 53 PM

3. Cursor is not pointer on error messages

https://user-images.githubusercontent.com/8112138/144947955-99d782e1-6091-4d34-874c-c7a3aaee83cb.mov


https://user-images.githubusercontent.com/8112138/144947969-10c30628-7352-4bee-a7a0-31dd05ee7eed.mov

**Comment by danjm** [comment]
@darkwing @georgewrmarshall @adonesky1 I just pushed one more commit that fixes the last issue to do with transaction insights "Swaps and Simple Send txs have 404 error code in Transaction Data in Tx details."

**Review by georgewrmarshall — APPROVED** [review]
lgtm

**Review by darkwing — APPROVED** [review]

**Pull request merged**